generate/generate.go

```diff
@@ -21,7 +21,14 @@ var (
 
 // Generator represents a generator for a container spec.
 type Generator struct {
-	spec *rspec.Spec
+	spec       *rspec.Spec
+	ExportOpts *ExportOptions
```

Comment:
Can we just pass this to the `Save*` functions?  I think this is [per-call enough][1] to not want a Generator-wide setting.

[1]: https://github.com/opencontainers/ocitools/pull/138#discussion_r71803604